rjfoley/style.css

```diff
@@ -0,0 +1,23 @@
+/* Import fonts */
+
+/* Re-styling Basics i.e.: body h1 h2 h3 ... ; */
+
+/* Snippet Title */
+
+.only-text {
+	font-size:24px;
+	line-height: 1.2em;
+	font-family: "Helvetica Neue";
+	max-width: 720px
```

Comment:
Decided to take a peak as well. Missing semi-colon after this max-width.

Reply:
Aha! You caught it.

Reply:
Good catch! Totally missed that.